### DISCUSSION TITLE
400: Bad Request - Email Alerts

### DISCUSSION THREAD

**Discussion by aleksmastermind2** [amidaware/tacticalrmm#1249] [General]
**Server Info (please complete the following information):**
 - OS: [Debian 5.10.136-1]
 - Browser: [Firefox 103.0.2]
 - RMM Version (0.14.6):

**Installation Method:**
  - [*] Standard

**Agent Info (please complete the following information):**
- Agent version (2.3.0):
- Agent OS: [Windows 11 Pro, 64 bit v21H2 (build 22000.194)]

**Describe the bug**
This bug is not attached to the agent and has nothing to do with it.
The problem is setup of email alerts.
We are getting "400: Bad Request" constantly, with other email addresses and servers.
Ports we tried are 587 and 465 with multiple different email server combinations.
Tried yandex, our server, hotmail...

**To Reproduce**
1. Go to 'Settings'
2. Click on 'Global settings'
3. Click on 'Email alerts'
4. Type the SMTP settings

**Expected behavior**
Expected green color with success message :)

**Screenshots**
<img width="619" alt="Capture" src="https://user-images.githubusercontent.com/81482222/185788512-7749f100-04c3-4572-ae05-a34ad8e61416.PNG">

**Additional context**
<3

**Comment by dinger1986** [comment]
Converted to discussion as this is an issue with your setup.

Where is it hosted? 

Have you put in the login details or is it an open smtp relay server?

**Reply by aleksmastermind2** [reply]
Linode
Yes, authentication is required.

**Reply by dinger1986** [reply]
I presume you have read this and actioned?

https://www.linode.com/community/questions/19082/i-just-created-my-first-linode-and-i-cant-send-emails-why

**Reply by wh1te909** [reply]
yep you need to ticket Linode and ask them to unblock those ports

**Reply by aleksmastermind2** [reply]
Thanks for the link.
Didn't knew about that.

I changed the DNS-s and i will post update tomorrow.
Most probably that Linode is the problem.

Thanks guys ✌️

**Reply by aleksmastermind2** [reply]
Hello :)

Just confirming that Linode was the issue.
Its working great.

Thanks for help.

**Reply by dinger1986** [reply]
perfect

**Reply by 3gmgithub** [reply]
I found that I could not use an eMail server with the SSL settings, so non-secure email worked.
Has anyone else figured out how to setup secure email?

**Reply by dinger1986** [reply]
What host?

I wrote and tested the below link and not had any issues or heard anyone else reporting issues.
https://docs.tacticalrmm.com/functions/email_alert/#email-setup

**Reply by 3gmgithub** [reply]
Inmotionhosting.  Looks like they also do not provide correct eMail header information.
I switched to using MailGun and that works perfectly.

**Reply by silversword411** [reply]
Inmotionhosting sucks, they've outsourced the  outbound mail stream to a 3rd party filtering service to try and "fix" their crappy servers and poorly secured customer shared hosting servers to try and minimize the outbound spam flow. This is so their shared host IPs don't keep getting added to ORBS lists and have one customer affect another customers email. But it does, and they can't fix the problem. And I think their network has been compromised, and they are fighting attackers in their infra on top of it all.

It's cheap, so there's not enough engineering devoted to their products.
